src/apps/relay/dbdrivers/dbd_redis.c

```diff
@@ -134,9 +134,9 @@ static Ryconninfo *RyconninfoParse(const char *userdb, char **errmsg) {
       } else if (!strcmp(s, "secret")) {
         co->password = strdup(seq + 1);
       } else if (!strcmp(s, "port")) {
-        co->port = (unsigned int)atoi(seq + 1);
```

Comment:
I recommend caution in changing the dbdrivers code. It's not guaranteed that a database protocol uses the same underlying concept of a port that the TCP and UDP protocols use.

Reply:
You might want to defer any changes to the dbdrivers to another pull request in the future.